### DISCUSSION TITLE
The DualSense controller stopped working since the app upgrade.

### DISCUSSION THREAD

**Discussion by Chakibdev** [LizardByte/community#415] [Sunshine]
Hi Sunshine team, I just want to report that since the upgrade from version 0.20.0 to version 0.21.0, the DualSense controller is no longer working with Alice: Madness Returns. I had to revert back to the previous version, 0.20.0, to make it work. 

I have also tried using the latest update, version 0.22.2, but the DualSense controller stopped working when I opened any game. I'm using the Windows 11 operating system and an Nvidia Shield 2019.

Thank you

**Comment by ReenigneArcher** [comment]
Please open a bug report with details that can help to reproduce your issue.

**Comment by FallingSnow** [comment]
I'm on Sunshine version 0.22.0 and my PS5 controller works as expected using Moonlight-tv playing Ratchet and Clank.

[a comment by an automated account is not shown]
